jaeger_client/reporter.py

```diff
@@ -79,7 +79,7 @@ class Reporter(NullReporter):
     """Receives completed spans from Tracer and submits them out of process."""
     def __init__(self, channel, queue_capacity=100, batch_size=10,
                  flush_interval=DEFAULT_FLUSH_INTERVAL, io_loop=None,
-                 error_reporter=None, metrics=None, **kwargs):
+                 error_reporter=None, metrics_factory=None, **kwargs):
```

Comment:
this is not backwards compatible. Reporter is a public class, so better to support the old `metrics` kv as well.